nvflare/fuel/hci/client/cli.py

```diff
@@ -444,7 +444,9 @@ def print_resp(self, resp: dict):
 
         if "data" in resp:
             for item in resp["data"]:
-                item_type = item["type"]
+                if not isinstance(item, dict):
+                    continue
+                item_type = item.get("type")
```

Comment:
just curious what other types can the item be?
strings or lists?

Reply:
string, table, error or dict.